frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -320,6 +320,19 @@ export const getParameters = createSelector(
   },
 );
 
+export const getRequiredParameters = createSelector(
```

Comment:
I don't fully understand the idea of having two selectors and using only one. 

Is it some kind of memoization/optimization? I double there will be more than 20 dashboard parameters ever

Reply:
My idea was to have a separate selector to get all required parameters when needed, but you're right, I'll remove it as it's not needed.